Jenkinsfile

```diff
@@ -57,6 +57,29 @@ pipeline {
         always {
             junit '**/target/test-reports/*'
         }
+        success {
+            script {
+                def vaultPath = 'secret/dsde/monster/dev/codecov/monster-sbt-plugins'
+                def codecov = [
+                        '1>&2 bash <(curl -s https://codecov.io/bash)',
+                        '-K',
+                        '-t ${CODECOV_TOKEN}',
```

Comment:
CODECOV token is required because it's a service we pay for and must auth for it?

Reply:
It's the UUID for the org/repo. I'm honestly not sure how codecov makes money, they might charge for private GitHub repos.